From 98062844f4bf1642d16e1854be5bca5bd478e371 Mon Sep 17 00:00:00 2001
From: Lee Wells <lee_wells@jabil.com>
Date: Wed, 11 Oct 2023 09:17:54 -0400
Subject: [PATCH] Use standard libarchive instead of the custom
 libarchve-luxonis

Having 2 packages in yocto trying to install the same headers/libs
won't work.  So just use the standard libarchive that yocto provides,
instead of libarchive-luxonis
---
 CMakeLists.txt                  | 2 +-
 cmake/depthaiDependencies.cmake | 3 ---
 2 files changed, 1 insertion(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ffea35ef..872b25b6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -413,7 +413,7 @@ target_link_libraries(${TARGET_CORE_NAME}
         Threads::Threads
         bz2
         FP16::fp16
-        archive_static
+        -larchive
         spdlog::spdlog
         z
 )
diff --git a/cmake/depthaiDependencies.cmake b/cmake/depthaiDependencies.cmake
index 0683601b..b7d1cb00 100644
--- a/cmake/depthaiDependencies.cmake
+++ b/cmake/depthaiDependencies.cmake
@@ -21,9 +21,6 @@ if(NOT CONFIG_MODE OR (CONFIG_MODE AND NOT DEPTHAI_SHARED_LIBS))
     # FP16 for conversions
     find_package(FP16 ${_QUIET} CONFIG REQUIRED)
 
-    # libarchive for firmware packages
-    find_package(archive_static ${_QUIET} CONFIG REQUIRED)
-
     # ZLIB for compressing Apps
     find_package(ZLIB REQUIRED)
 
-- 
2.34.1

